### DISCUSSION TITLE
Button & Card components

### DISCUSSION THREAD

**Discussion by vitalijalbu** [chakra-ui/ark#2972] [General]
Hi guys, will be nice to have also these 2 🚀💪

**Comment by segunadebayo** [comment]
No plans to add a Button or Card component. Those are purely UI or presentation components.

**Reply by vitalijalbu** [reply]
but we need them, they are essential

**Comment by davidmoshal** [comment]
I'm not following, how does one implement a button, with hover, click, etc states, and button groups?
